From: Jonathan Nieder <jrnieder@gmail.com>
Date: Mon, 25 Oct 2010 03:14:30 -0500
Subject: Set liblzma version-info to 2:0:0

This change is Debian-specific.

The upstream version-info is 5:0:0, but Debian has been
using version 2 for a while and it is simplest to avoid
breaking ABI.

With commit 418d64a (Fix a design error in liblzma API,
2009-11-14) the lzma_options_lzma data structure was changed.
Programs that read or write its fields will have no hope of
working with older liblzma versions.

With commit 1a7ec87 (Revised the Index handling code.,
2009-12-31) various functions were removed.  Programs that
used them will not work with the current library, so that
required bumping the soname for a second time.

Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 src/liblzma/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/liblzma/Makefile.am b/src/liblzma/Makefile.am
index 6d05fc6..ee34c83 100644
--- a/src/liblzma/Makefile.am
+++ b/src/liblzma/Makefile.am
@@ -24,7 +24,7 @@ liblzma_la_CPPFLAGS = \
 	-I$(top_srcdir)/src/liblzma/simple \
 	-I$(top_srcdir)/src/common \
 	-DTUKLIB_SYMBOL_PREFIX=lzma_
-liblzma_la_LDFLAGS = -no-undefined -version-info 5:0:0
+liblzma_la_LDFLAGS = -no-undefined -version-info 2:0:0
 
 include $(srcdir)/common/Makefile.inc
 include $(srcdir)/check/Makefile.inc
-- 
1.7.2.3

